microservice_demo/README.md

```diff
@@ -0,0 +1,36 @@
+# Kernel Gateway Microservice Demonstration
+
+The [Kernel Gateway](https://github.com/jupyter-incubator/kernel_gateway) is a
+[JupyterApp](https://github.com/jupyter/jupyter_core/blob/master/jupyter_core/application.py) that
+implements different APIs and protocols for accessing Jupyter kernels.  We can use the Jupyter kernel gateway to
+[process requests](https://github.com/jupyter-incubator/kernel_gateway#processing-requests) and transform an
+annotated notebook into a HTTP API using the Jupyter kernel gateway.
+ 
+This example shows how one can use the Jupyter kernel gateway to deploy an notebook as a
+microservice at the heart of a Twitter bot, using a combination of [Alchemy API](http://www.alchemyapi.com/products/alchemylanguage/entity-extraction) with IFTTT's [Twitter](https://ifttt.com/twitter) and [Maker](https://ifttt.com/maker) channels, to respond to mentions with a search URL for relevant Meetups. You will need developer keys for [Alchemy](http://www.alchemyapi.com/api/register.html) and [Maker](https://ifttt.com/maker) to proceed, as well as an arbitrary Event Name for use with Maker.
+
+ ### Build the docker container
+ ```
+ make build
+ ```
+
+ ### Run the docker container for the service
+>After having set the ALCHEMY_KEY, MAKER_KEY, and MAKER_EVENT_NAME environment variables accordingly. These variables are passed into the docker container, read and used by the notebook, and must still be set if running the Jupyter kernel gateway outside of a container.
+
+ ```
+ make run
```

Comment:
Show the env vars getting set in the code block. You can do it right on the run command like

```
make run ALCHEMY_KEY=your_key
```

and so on for the rest.